src/components/buttons.js

```diff
@@ -1,29 +1,77 @@
 import React from 'react';
+import '../App.css';
+import PropTypes from 'prop-types';
+
+function Buttons({ handleCalculation }) {
+  const handleClick = (buttonName) => {
+    handleCalculation(buttonName);
+  };
 
-function Buttons() {
   return (
     <div className="buttons">
-      <button className="button" type="button">AC</button>
-      <button className="button" type="button">+/-</button>
-      <button className="button" type="button">%</button>
-      <button className="button operator" type="button">/</button>
-      <button className="button" type="button">7</button>
-      <button className="button" type="button">8</button>
-      <button className="button" type="button">9</button>
-      <button className="button operator" type="button">X</button>
-      <button className="button" type="button">4</button>
-      <button className="button" type="button">5</button>
-      <button className="button" type="button">6</button>
-      <button className="button operator" type="button">-</button>
-      <button className="button" type="button">1</button>
-      <button className="button" type="button">2</button>
-      <button className="button" type="button">3</button>
-      <button className="button operator" type="button">+</button>
-      <button className="button ziro" type="button">0</button>
-      <button className="button" type="button">.</button>
-      <button className="button operator" type="button">=</button>
+      <button className="button" type="button" onClick={() => handleClick('AC')}>
+        AC
+      </button>
+      <button className="button" type="button" onClick={() => handleClick('+/-')}>
+        +/-
+      </button>
+      <button className="button" type="button" onClick={() => handleClick('%')}>
+        %
+      </button>
+      <button className="button operator" type="button" onClick={() => handleClick('/')}>
+        /
+      </button>
```

Comment:
- [ ] When I try to make a divide operation the error in the image below is raised, Please fix that by replacing the slash `/` with the divide symbol `÷`.

![image](https://github.com/otmaneechchafyky/MathMagicians/assets/68796232/80ab6f85-f6b2-4672-bdae-992126839bc3)

Reply:
Okay, I am on it. I just forgot to modify it.